integration-tests/relayinterface/chain_components_test.go

```diff
@@ -94,129 +98,121 @@ func DisableTests(it *SolanaChainComponentsInterfaceTester[*testing.T]) {
 	})
 }
 
-func RunChainComponentsSolanaTests[T TestingT[T]](t T, it *SolanaChainComponentsInterfaceTester[T]) {
+func RunChainComponentsSolanaTests[T WrappedTestingT[T]](t T, it *SolanaChainComponentsInterfaceTester[T]) {
 	RunContractReaderSolanaTests(t, it)
 	// Add ChainWriter tests here
 }
 
-func RunChainComponentsInLoopSolanaTests[T TestingT[T]](t T, it ChainComponentsInterfaceTester[T]) {
+func RunChainComponentsInLoopSolanaTests[T WrappedTestingT[T]](t T, it ChainComponentsInterfaceTester[T]) {
 	RunContractReaderInLoopTests(t, it)
 	// Add ChainWriter tests here
 }
 
-func RunContractReaderSolanaTests[T TestingT[T]](t T, it *SolanaChainComponentsInterfaceTester[T]) {
+func RunContractReaderSolanaTests[T WrappedTestingT[T]](t T, it *SolanaChainComponentsInterfaceTester[T]) {
 	RunContractReaderInterfaceTests(t, it, false, true)
 
 	var testCases []Testcase[T]
 
 	RunTests(t, it, testCases)
 }
 
-func RunContractReaderInLoopTests[T TestingT[T]](t T, it ChainComponentsInterfaceTester[T]) {
+func RunContractReaderInLoopTests[T WrappedTestingT[T]](t T, it ChainComponentsInterfaceTester[T]) {
 	RunContractReaderInterfaceTests(t, it, false, true)
 
 	var testCases []Testcase[T]
 
 	RunTests(t, it, testCases)
 }
 
-type SolanaChainComponentsInterfaceTesterHelper[T TestingT[T]] interface {
+type SolanaChainComponentsInterfaceTesterHelper[T WrappedTestingT[T]] interface {
 	Init(t T)
 	RPCClient() *chainreader.RPCClientWrapper
 	Context(t T) context.Context
 	Logger(t T) logger.Logger
-	GetJSONEncodedIDL(t T) []byte
-	CreateAccount(t T, value uint64) solana.PublicKey
+	GetPrimaryIDL(t T) []byte
+	GetSecondaryIDL(t T) []byte
+	CreateAccount(t T, it SolanaChainComponentsInterfaceTester[T], contractName string, value uint64, testStruct TestStruct) solana.PublicKey
+	TXM() *txm.TxManager
+	SolanaClient() *client.Client
+}
+
+type WrappedTestingT[T any] interface {
```

Comment:
I could have made this change in `chainlink-common` so I wouldn't have to wrap here but wanted to avoid making changes in another repo.